src/azure-cli/HISTORY.rst

```diff
@@ -26,6 +26,7 @@ Release History
 * Fix issue #7154: Updating documentation for command <> to use back ticks instead of single quotes
 * Fix issue #11287: webapp up: By default make the app created using up 'should be 'SSL enabled'
 * Fix issue #11592: Add az webapp up flag for html static sites
+* functionapp: Add ability to create Java function apps in Linux
```

Comment:
Please place your release note in the right position, which should be above 2.0.79

Reply:
Thanks, @Juliehzl, just fixed.